consumer/pull_consumer.go

```diff
@@ -193,6 +201,14 @@ func (pc *defaultPullConsumer) Poll(ctx context.Context, timeout time.Duration)
 	case <-ctx.Done():
 		return nil, ErrNoNewMsg
 	case cr := <-pc.consumeRequestCache:
+		if cr.GetPQ().IsDroppd() {
+			rlog.Info("defaultPullConsumer poll the message queue not be able to consume, because it was dropped", map[string]interface{}{
```

Comment:
`GetPQ()` is unnecessary. Use `cr.processQueue` may be better.

Reply:
okay,but I hope expose the pq and mq because I want to check the consume status sometimes.